src/ast/visitor.rs

```diff
@@ -124,6 +124,10 @@ visit_noop!(bigdecimal::BigDecimal);
 /// node and `post_visit_` methods are invoked after visiting all
 /// children of the node.
 ///
+/// Important note: The `Break` type should be kept as small as possible to prevent
```

Comment:
could we place the note on the documentation of `Break` instead? [here for example](https://github.com/niebayes/datafusion-sqlparser-rs/blob/6419cbd073d6fda4f8b43e7b320d9642ae1bba66/src/ast/visitor.rs#L300)